### DISCUSSION TITLE
Can a slant Line counter work?

### DISCUSSION THREAD

**Discussion by yeongnamtan** [roboflow/supervision#597] [Q&A]
### Search before asking

- [X] I have searched the Supervision [issues](https://github.com/roboflow/supervision/issues) and found no similar feature requests.

### Question

Can line counter work if the line is not vertical or horizontal and is slanted ?

**Comment by SkalskiP** [comment]
Hi @yeongnamtan! 👋🏻 Thanks for your interest in supervision. Let me start by converting this issue into a discussion and moving it into Q&A section.

**Comment by SkalskiP** [comment]
Hi @yeongnamtan! The slant line counter should work. Did you face any issues?
